sectionproperties/analysis/fea.py

```diff
@@ -343,15 +407,43 @@ def plastic_properties(self, u, p):
             force += gp[0] * j * self.material.yield_strength
 
         # calculate element centroid
+        # try:
```

Comment:
Can this commented out try statement be removed?

Reply:
Yep

Reply:
Removed!